src/main/features/claimant-response/views/check-and-send.njk

```diff
@@ -11,47 +11,89 @@
 
     {{ tableStart('yourResponse', 'Your response') }}
 
-    {% if draft.settleAdmitted %}
-      {% if draft.settleAdmitted.admitted.option === 'yes' %}
-        {% set settleAdmittedResponse = 'I accept this amount' %}
-      {% else %}
-        {% set settleAdmittedResponse = 'I reject this amount' %}
+    {% if alreadyPaid %}
+
+      {% if draft.partPaymentReceived %}
+        {% set paymentReceivedResponse = 'Yes' if draft.partPaymentReceived.received.option === 'yes' else 'No' %}
+
+        {{ row(label = 'Do you agree the defendant has paid ' + amount | numeral + '?', bold = true,
+          value = 'Yes' if draft.partPaymentReceived.received.option === 'yes' else 'No',
+          changeLink = ClaimantResponsePaths.partPaymentReceivedPage.evaluateUri({ externalId: claim.externalId }), bottomBorder = true) }}
+      {% endif %}
+
+      {% if draft.accepted %}
+        {% if draft.accepted.accepted.option === 'yes' %}
+          {% set paymentAcceptedResponse = 'Yes' %}
```

Comment:
`| renderYesNo` here and all other places as well please.

Reply:
YesNoViewFilter takes boolean input, using it here would result in it always rendering 'Yes'

Reply:
If we create a new filter  for `YesNoOption` and use that, it will make code readable.